index.js

```diff
@@ -456,17 +461,38 @@
   //# Any :: Type
   //.
   //. Type comprising every JavaScript value.
-  var Any = NullaryTypeWithUrl('sanctuary-def/Any', K(true));
+  var Any = {
+    type: NULLARY,
+    name: 'sanctuary-def/Any',
+    url: functionUrl('sanctuary-def/Any'),
+    constructor: _Type,
+    test: K(true),
+    validate: Right,
+    format: function(outer) { return outer('Any'); },
+    keys: [],
+    types: {},
+    toString: K('Any')
+  };
+  Any.parent = Any;
+
+  //# None :: Type
+  //.
+  //. Type with no members.
+  var None = NullaryTypeWithUrl(Any, 'sanctuary-def/None', K(false));
```

Comment:
I'm not sure if we need this. I added it to use in INCONSISTENT only, but we could as well use `Any` in there since the `_test` is `K(false)`.